cfc-testing/performance-synth-benchmark.sh

```
@@ -0,0 +1,251 @@
+    local deployment_option="$3"
+    local description="$4"
+    
+    echo -e "${YELLOW}📊 Running benchmark: $test_name${NC}"
+    echo -e "${YELLOW}Description: $description${NC}"
+    echo -e "${YELLOW}Stack: $stack_name, Option: $deployment_option${NC}"
+    
+    # Clean up any existing context
+    rm -f deployment-context.json
+    
+    # Measure synthesis time
+    local start_time=$(date +%s.%N)
```

Shellcheck (0.11.0)

[warning] Declare and assign separately to avoid masking return values.

(SC2155)